src/system/SearchModule/Api/UserApi.php

```diff
@@ -229,11 +238,11 @@ public function getall($args)
         if (!isset($args['numitems']) || !is_numeric($args['numitems'])) {
             $args['numitems'] = -1;
         }
-        if (!isset($args['sortorder']) || !in_array($args['sortorder'], array('count', 'date'))) {
+        if (!isset($args['sortorder']) || !in_array($args['sortorder'], ['count', 'date'])) {
             $args['sortorder'] = 'count';
         }
 
-        $items = array();
+        $items = [];
 
         // Security check
         if (!SecurityUtil::checkPermission('ZikulaSearchModule::', '::', ACCESS_OVERVIEW)) {
```

Comment:
I didn't refactor the api yet on purpose for two reasons: 1st it is quite difficult to incorporate old search plugins and 2nd #1753 is about Twig migration. So my primary focus is on getting things converted to Twig and Sf Forms.